packages/electrode-redux-router-engine/lib/util.js

```diff
@@ -10,5 +14,14 @@ module.exports = {
     if (p.startsWith("/")) return p;
 
     return p.startsWith(".") ? Path.resolve(baseDir || "", p) : p;
+  },
+  getExtractor: () => {
+    if (process.env.NODE_ENV === "production" && extractor) return extractor;
```

Comment:
It may be cleaner if the function name `getExtractor` is called `createdStatExtractor` and it doesn't deal with memoization for `production`.  Instead, memoization should be handled by caller.